docs/conf.py

```diff
@@ -163,3 +164,16 @@
 html_sourcelink_suffix = ''
 
 autoclass_content = 'both'
+
+# -- Redirects ---------------------------------------------------------------
+
+rediraffe_redirects = {
+    "tutorials/chsh_with_estimator.ipynb": "tutorials/chsh-with-estimator/index.ipynb",
+    "tutorials/Error-Suppression-and-Error-Mitigation.ipynb": "tutorials/error-suppression-and-error-mitigation/index.ipynb",
+    "tutorials/grover_with_sampler.ipynb": "tutorials/grover-with-sampler/index.ipynb",
+    "tutorials/how-to-getting-started-with-estimator.ipynb": "tutorials/how-to-getting-started-with-estimator/index.ipynb",
+    "tutorials/how-to-getting-started-with-sampler.ipynb": "tutorials/how-to-getting-started-with-sampler/index.ipynb",
+    "tutorials/qaoa_with_primitives.ipynb": "tutorials/qaoa-with-primitives/index.ipynb",
+    "tutorials/user-transpiled-circuits.ipynb": "tutorials/user-transpiled-circuits/index.ipynb",
+    "tutorials/vqe_with_estimator.ipynb": "tutorials/vqe-with-estimator/index.ipynb"
```

Comment:
This will be more readable if you use a dictionary comprehension to create the value. 

```python
{f"tutorials/{name}.ipynb": "tutorials/{name}/index.ipynb" for name in ["chsh_with_estimator", ...]}
```

Reply:
Nice, thanks!